akka-distributed-data/src/main/scala/akka/cluster/ddata/protobuf/ReplicatedDataSerializer.scala

```diff
@@ -677,22 +677,19 @@ class ReplicatedDataSerializer(val system: ExtendedActorSystem)
 
   private def ormapDeltaGroupToProto(deltaGroup: ORMap.DeltaGroup[_, _]): rd.ORMapDeltaGroup = {
```

Comment:
could we change the signature of this to something like:

    private def ormapOpsToProto(ops: immutable.Seq[ORMap.DeltaOp]): rd.ORMapDeltaGroup

and then the above methods don't have to create the DeltaGroup

    private def ormapUpdateToProto(deltaOp: ORMap.UpdateDeltaOp[_, _]): rd.ORMapDeltaGroup =
      ormapOpsToProto(List(deltaOp))